test/libsolidity/syntaxTests/tight_packing_literals_050.sol

```diff
@@ -1,34 +1,9 @@
 pragma experimental "v0.5.0";
```

Comment:
One uses `pragma experimental "v0.5.0"` and the other does not. In addition to that, the expectation is different, so I don't see why it can be removed yet.